File: VB/MVVMDemo.Wpf/Modules/DXCommand/MultipleCallsViewModel.vb
Imports DevExpress.Mvvm
Imports System.Windows

Namespace MVVMDemo.DXCommandDemo
    Public Class MultipleCallsViewModel
        Inherits BindableBase

        Public Property IsReadonly() As Boolean
            Get
                Return GetProperty(Function() IsReadonly)
            End Get
            Set(ByVal value As Boolean)
                SetProperty(Function() IsReadonly, value)
            End Set
        End Property

        Public Property FileName() As String
            Get
                Return GetProperty(Function() FileName)
            End Get
            Set(ByVal value As String)
                SetProperty(Function() FileName, value)
            End Set
        End Property

        Public Sub Open()
            MessageBox.Show(String.Format("Action: Open {0}", FileName))
        End Sub
        Public Function CanOpen() As Boolean
            Return Not String.IsNullOrEmpty(FileName)
        End Function

        Public Sub Clear()
            FileName = Nothing
        End Sub
    End Class
End Namespace

File: CS/MVVMDemo.Wpf/Modules/DXCommand/MultipleCallsViewModel.cs
using DevExpress.Mvvm;
using System.Windows;

namespace MVVMDemo.DXCommandDemo {
    public class MultipleCallsViewModel : BindableBase {
        public bool IsReadonly {
            get { return GetProperty(() => IsReadonly); }
            set { SetProperty(() => IsReadonly, value); }
        }

        public string FileName {
            get { return GetProperty(() => FileName); }
            set { SetProperty(() => FileName, value); }
        }

        public void Open() {
            MessageBox.Show(string.Format("Action: Open {0}", FileName));
        }
        public bool CanOpen() {
            return !string.IsNullOrEmpty(FileName);
        }

        public void Clear() {
            FileName = null;
        }
    }
}
